crates/astria-core/src/sequencer/mod.rs

```diff
@@ -1 +1,39 @@
 pub mod v1;
+
+/// A trait to convert from raw decoded protobuf types to idiomatic astria types.
+///
+/// The primary use of this trait is to convert to/from foreign types.
+pub trait Protobuf: Sized {
```

Comment:
I had to move the Protobuf trait into the crates that were implementing it - the problem is that it was implemented on the external `merkle::Proof` type, however we then had two implementations of it (one in `sequencer` and one in `sequencerblock`) which conflicted as the `Raw` types weren't the same.

Reply:
Ok, that's not good. We have to find a different solution because that's pretty confusing then. I really don't want to duplicate traits like this.